ssl/ssl_cipher.cc

```diff
@@ -1336,9 +1336,11 @@ bool ssl_create_cipher_list(UniquePtr<SSLCipherPreferenceList> *out_cipher_list,
 
   *out_cipher_list = std::move(pref_list);
 
-  // Configuring an empty cipher list is an error but still updates the
-  // output.
-  if (sk_SSL_CIPHER_num((*out_cipher_list)->ciphers.get()) == 0) {
+  // Configuring an empty cipher list is an error with |strict| but still
+  // updates the output. When otherwise, OpenSSL explicitly allows an empty
+  // list.
+  if ((strict || (*rule_str != '\0')) &&
```

Comment:
NP: comment slight confusing, maybe just "Otherwise, OpenSSL explicitly..."

Reply:
"When otherwise" was meant when "strict is not true", so I don't think that clarifies things better. I'll change the previous sentence and see if that helps